jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -109,7 +109,7 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmp_path):
     )
 
     result = dc[-1].get_object(cls=CCDData)
-    assert result.shape == (4, 2)  # Cube shape is (2, 2, 4)
+    assert result.shape == (2, 4)  # Cube shape is (2, 2, 4), moment transposes
```

Comment:
What does "moment transposes" mean? Is it supposed to?

Reply:
Yes, apparently it needs it for the output to display correctly (see screenshot below for proof that it matches the input), I think I just added this comment to remind myself why it was the opposite of what I expected while updating the tests. I can't explain off the top of my head why that's needed I just see that it comes out right.

<img width="1006" alt="Screenshot 2024-08-05 at 2 35 40 PM" src="https://github.com/user-attachments/assets/131f1c15-1c61-4d98-b38b-eb4952a5a5fe">

Reply:
Does this confusing behavior go away with specutils 2.0? If not, I think we need to understand why.

Reply:
I think it's just a matter of what we want on x and y (RA vs dec) and array axis order vs display axis order. I agree it would be better to understand/remember in more detail, but it has nothing to do with this PR.